src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -313,6 +343,13 @@ static void uct_ib_mlx5_wqe_dump(uct_ib_iface_t *iface, void *wqe, void *qstart,
                            s, ends - s);
     s += strlen(s);
 
+#if HAVE_MLX5_MMO
+    if (opcode == MLX5_OPCODE_MMO) {
+        uct_ib_mlx5_dump_dma_wqe(wqe, s, ends - s);
+        return;
+    }
+#endif
```

Comment:
can we unitfy the code of printing DMA/GGA wqe with the existing code flow? instead of making a special case for it.
Seems the only "special" thing about GGA is that it has DMA segment. So it can be treated same way as atomic/rdma segments for example.